components/data-bridge/org.wso2.carbon.databridge.streamdefn.filesystem/src/main/java/org/wso2/carbon/databridge/streamdefn/filesystem/internal/EventStreamListenerImpl.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.databridge.streamdefn.filesystem.internal;
```

Comment:
@sinthuja 
If we move it to databridge core, we will have to make the event stream core a compile time dependency for databridge core. I think the original design for FileBasedStreamDefnStore was to keep event stream dependency only at FileBasedStreamDefnStore and not at databridge core level. If we move it to the core, the databridge core and event stream core would become coupled more tightly IMHO. Therefore shall we keep the dependency at FileBasedStreamDefnStore itself? WDYT?